### PR TITLE
Feature/dark mode

### PR DESCRIPTION
**Pull request by alcipir**

### CODE REVIEW

File: src/components/Settings.vue [hunk 1]
```diff
@@ -17,6 +17,17 @@
             label="Networking"
           />
         </q-tabs>
+          <q-tabs
```
Comment (schancel) [inline src/components/Settings.vue:20]: Should there be another tabs group here?  Seems odd.

File: src/components/dialogs/SettingsDialog.vue [hunk 2]
```diff
@@ -37,16 +37,22 @@ export default {
       settings: {
         networking: {
           updateInterval: this.getUpdateInterval() / 1_000
+        },
+        appearance: {
+          darkMode: this.getDarkMode()
```
Comment (schancel) [inline src/components/dialogs/SettingsDialog.vue:42]: Is this just a toggle right now?

File: src/pages/Chat.vue [hunk 3]
```diff
@@ -8,7 +8,7 @@
     <q-scroll-area
       ref="chatScroll"
       class="q-px-md row"
-      :style="`height: calc(100% - ${offsetHeight()}px); min-height: calc(100%-${offsetHeight()}px); background: lightblue`"
+      :style="`height: calc(100% - ${offsetHeight()}px); min-height: calc(100%-${offsetHeight()}px); background: ${bodyBackground}`"
```
Comment (schancel) [inline src/pages/Chat.vue:11]: Is it possible to actually create a CSS theme to avoid passing all this stuff around?

File: src/store/modules/my_profile.js [hunk 4]
```diff
@@ -22,6 +22,9 @@ export default {
     setRelayData (state, relayData) {
       state.profile = relayData.profile
       state.inbox = relayData.inbox
+    },
```
Comment (schancel) [inline src/store/modules/my_profile.js:25]: This looks unrelated to the PR topic and should probably be a separate PR.  Thank you for fixing this though!

**Comment by schancel** [comment]
Awesome!  Thank you.  Just a few questions -- I'll answer some of them myself by trying it out.

File: src/components/Settings.vue (hunk 1 above)
Comment (alcipir) [reply src/components/Settings.vue]: fixed

File: src/pages/Chat.vue (hunk 3 above)
Comment (alcipir) [reply src/pages/Chat.vue:11]: definitely, I will see what I can do

File: src/store/modules/my_profile.js (hunk 4 above)
Comment (alcipir) [reply src/store/modules/my_profile.js:25]: This is odd, I think I got this from another commit?

File: src/components/dialogs/SettingsDialog.vue (hunk 2 above)
Comment (alcipir) [reply src/components/dialogs/SettingsDialog.vue:42]: Can you explain what do you mean by this? It is a boolean/toggle option. 

I think there is probably a better way to do this, but not sure right now what it is.

Comment (schancel) [reply src/components/dialogs/SettingsDialog.vue:42]: I was just thinking it would be the name of a theme or something, but this is fine for now.

File: src/store/modules/my_profile.js [hunk 5]
```diff
@@ -33,6 +36,9 @@ export default {
     },
     setRelayData ({ commit }, relayData) {
       commit('setRelayData', relayData)
+    },
```
Comment (schancel) [inline src/store/modules/my_profile.js:39]: I'm not sure where these changes keep coming from.  I think I had the same thing happen on my branch.  This should be removed from this PR though, then we can merge.

Comment (alcipir) [reply src/store/modules/my_profile.js]: done
